pom.xml

```diff
@@ -68,10 +68,15 @@
       <version>1.2.17</version>
     </dependency>
     <dependency>
-      <groupId>com.facebook.hadoop</groupId>
-      <artifactId>hadoop-core</artifactId>
-      <version>0.20.0</version>
-    </dependency>
+        <groupId>org.apache.hadoop</groupId>
+        <artifactId>hadoop-core</artifactId>
+        <version>0.20.2-cdh3u6</version>
+    </dependency>
+    <!--dependency>
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/90cd2436c4d64355eca954894e012f22299ec367